system/View/Table.php

```diff
@@ -83,6 +83,11 @@ class Table
      */
     public $function;
 
+    /**
+     * Order each inserted row by heading keys
+     */
+    public bool $rowKeysSyncWithHeadingKeys = false;
```

Comment:
Not a big fan of the variable name. Maybe something simpler, like: `syncRowsWithHeading` would be better?

Reply:
Great name, I will change it to that